src/demo_hook_async.c

```diff
@@ -27,6 +27,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <uiohook.h>
+#include <wchar.h
```

Comment:
#include <wchar.h**>**

Reply:
aw crap, I think I made this change in the online editor thinking it was so simple that there's no way I'd screw it up.

Reply:
Hehe just one more benefit of code review.